packages/calendar/react/app.tsx

```diff
@@ -11,7 +11,7 @@ if (!root) throw new Error("Root element not found");
 ReactDOM.createRoot(root).render(
   <React.StrictMode>
     <TRPCProvider>
-      <Calendar />
+      <Calendar onPageChange={(to) => (window.location.href = to)} />
```

Comment:
Mh? 

What's this good for?

Reply:
we're setting the location.href even though our extension uses an im-memory router? 🤔

Reply:
This app.tsx could be removed basically - we're never running the calendar as is.

Each consumer of the Calendar needs to send on a method on how it navigates.

On the calendar, we send on our `router.navigate`. This `app.tsx` is not being used but had to define it since I narrowed the type down (this prop was optional before which didn't make sense to me since it's necessary for the calendar to work properly)